packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js

```diff
@@ -16,8 +16,9 @@ import InputModalStepper from '../../containers/InputModalStepper';
 import Name from './Name';
 import Wrapper from './Wrapper';
 import Input from '../Input';
+import ErrorMessage from './ErrorMessage';
 
-const InputMedia = ({ label, onChange, name, attribute, value, type }) => {
+const InputMedia = ({ label, onChange, name, attribute, value, type, id, error }) => {
```

Comment:
The id and error props needs to be added to the `propTypes`.

Reply:
Regarding the comment about the baseline could you add the following in this file:

```
import { get, isEmpty } from 'lodash';
// ...

return (
  <Wrapper hasError={!isEmpty(error)}>
   // ...
  </Wrapper>
);
```